core-web/libs/sdk/client/src/lib/editor/sdk-editor.ts

```diff
@@ -43,6 +43,25 @@ export function editContentlet<T>(contentlet: Contentlet<T>) {
     });
 }
 
+/**
+ * Reorders the menu starting from a specified level.
+ *
+ * @param {number} [startLevel=1] - The level from which to start reordering the menu. Defaults to 1.
+ *
+ * This function constructs a URL for the reorder menu page with the given start level and a fixed depth of 2.
+ * It then sends a message to the editor with the action to reorder the menu and the constructed URL as the payload.
+ */
+export function reorderMenu(startLevel = 1) {
+    // This is the URL for the reorder menu page
+    // All params are hardcoded on the jsp, so here we just need to send the same URL
+    const reorderUrl = `/c/portal/layout?p_l_id=2df9f117-b140-44bf-93d7-5b10a36fb7f9&p_p_id=site-browser&p_p_action=1&p_p_state=maximized&_site_browser_struts_action=%2Fext%2Ffolders%2Forder_menu&startLevel=${startLevel}&depth=2`;
```

Comment:
The depth here? should be hardcoded? can we pass it?

Reply:
In traditional is hardcoded
But sure, i just add the depth as param here